### ISSUE TITLE
saltstack misreports version number

### ISSUE THREAD

**Issue by pedromaltez** [Homebrew/homebrew#43493]
````
salt --version
salt 2015.5.0 (Lithium)

salt --versions-report
           Salt: 2015.5.0
         Python: 2.7.10 (default, Jul 14 2015, 19:46:27)
         Jinja2: 2.7.3
       M2Crypto: 0.22
 msgpack-python: 0.4.6
   msgpack-pure: Not Installed
       pycrypto: 2.6.1
        libnacl: Not Installed
         PyYAML: 3.11
          ioflo: Not Installed
          PyZMQ: 14.7.0
           RAET: Not Installed
            ZMQ: 4.1.3
           Mako: Not Installed
        Tornado: Not Installed
        timelib: Not Installed
       dateutil: 1.5
````

This happens even though the actual version installed is 2015.5.3

This is because Homebrew currently uses the GitHub generated tarball to install salt. This was discussed in saltstack/salt#25307

The fix is to use the tarball manually attached to the release by the developers instead of the GitHub generated tarball. It is what we are instructed to do on the [relases page for salt][1].

[1]: https://github.com/saltstack/salt/releases (Releases - saltstack/salt)

**Comment by dunn** [comment]
Would you be able to open a pull request?  This should help https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md and we're happy to answer any questions!

**Comment by pedromaltez** [comment]
@dunn Yep, that was already in progress. :)

**Comment by dunn** [comment]
Awesome, thanks!

**Comment by pedromaltez** [comment]
Closed with #43495
